lib/cartopy/__init__.py

```diff
@@ -3,29 +3,6 @@
 # This file is part of Cartopy and is released under the LGPL license.
 # See COPYING and COPYING.LESSER in the root of the repository for full
 # licensing details.
-
-from ._version import version as __version__  # noqa: F401
-import tempfile
-
-__document_these__ = ['config']
-
-# Configuration
-import os.path
-
-# for the writable data directory (i.e. the one where new data goes), follow
-# the XDG guidelines found at
-# https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html
-_writable_dir = os.path.join(os.path.expanduser('~'), '.local', 'share')
-_data_dir = os.path.join(os.environ.get("XDG_DATA_HOME", _writable_dir),
-                         'cartopy')
-_cache_dir = os.path.join(tempfile.gettempdir(), 'cartopy_cache_dir')
-
-config = {'pre_existing_data_dir': os.environ.get('CARTOPY_DATA_DIR', ''),
-          'data_dir': _data_dir,
-          'cache_dir': _cache_dir,
-          'repo_data_dir': os.path.join(os.path.dirname(__file__), 'data'),
-          'downloaders': {},
-          }
 """
 The config dictionary stores global configuration values for cartopy.
```

Comment:
This is not a module docstring; it's a docstring for the `config` dictionary. I'm not sure it actually _works_ as a docstring though.

Reply:
OK, I agree. It doesn't do anything in the code i.e. `help(cartopy.config)` just tells me about a dict, but the documentation generated does use this. I've reverted these changes and removed that check from the pre-commit configuration.